modules/images-test-s2i.adoc

```diff
@@ -44,7 +44,7 @@ The S2I tooling comes with powerful generation tools to speed up the process of
 creating a new S2I image. The `s2i create` command produces all the necessary S2I
 scripts and testing tools along with the `Makefile`:
 
-[source,termnal]
+[source,terminal]
 ----
 $ s2i create _<image name>_ _<destination directory>_
```

Comment:
```suggestion
$ s2i create _<image_name>_ _<destination_directory>_
```
Currently, the user replaced values are not rendering in italics. I think adding the middle `_` will fix it but double check. Preview link: https://docs.openshift.com/container-platform/4.16/openshift_images/create-images.html#images-test-s2i-generating-scripts-and-tools_create-images

Reply:
You can disregard this comment. No need to push a fix for this after all. Sorry.